components/PositiveRateMixedChart.vue

```diff
@@ -365,11 +365,11 @@ const options: ThisTypedComponentOptionsWithRecordProps<
               ].toLocaleString()
               let label = `${
                 this.dataLabels[tooltipItem.datasetIndex!]
-              } : ${cases} ${unit}`
+              } : ${cases} 人`
```

Comment:
こちらも`人`が翻訳されないので、
```suggestion
              } : ${cases} ${this.$t('人')}`
```
とした方がよいです。

Reply:
こちらも合わせて修正いたします。